render/fx_renderer/fx_pass.c

```diff
@@ -343,8 +343,7 @@ void fx_render_pass_add_texture(struct fx_gles_render_pass *pass,
 
 	bool has_alpha = texture->has_alpha
 		|| alpha < 1.0
-		|| fx_options->corner_radius > 0
-		|| fx_options->discard_transparent;
```

Comment:
this an accidental removal or on purpose?

Reply:
Actually intentional; maybe best suited for a different PR, but the discarding of transparent pixels wouldn't make the texture itself transparent (it would already need to have transparent elements) so removing that line fixes a logical error